controller/projects.go

```diff
@@ -49,60 +48,69 @@ func NewProjects(
 
 func (p *Projects) Create(user *model.User, input model.NewProject) (*model.Project, error) {
 	proj := &model.Project{
-		ID:          uuid.New(),
-		Secret:      uuid.New(),
-		PublicID:    uuid.New(),
-		ParentID:    input.ParentID,
-		Seed:        input.Seed,
-		Title:       input.Title,
-		Description: input.Description,
-		Readme:      input.Readme,
-		Persist:     false,
-		Version:     p.version,
-		UserID:      user.ID,
+		ID:               uuid.New(),
+		Secret:           uuid.New(),
+		PublicID:         uuid.New(),
+		ParentID:         input.ParentID,
+		Seed:             input.Seed,
+		Title:            input.Title,
+		Description:      input.Description,
+		Readme:           input.Readme,
+		Persist:          false,
+		NumberOfAccounts: input.NumberOfAccounts,
+		Version:          p.version,
+		UserID:           user.ID,
 	}
 
-	ttpls := make([]*model.TransactionTemplate, len(input.TransactionTemplates))
-	for i, tpl := range input.TransactionTemplates {
-		ttpls[i] = &model.TransactionTemplate{
+	files := make(
+		[]*model.File,
+		len(input.TransactionTemplates)+len(input.ScriptTemplates)+len(input.ContractTemplates),
```

Comment:
In case that there's this complexity in length, is just not worth it defining the length, just define it as 0 and then append:
```suggestion
		files := make([]*model.File,0)
		files = append(files, &model.File{ ... })
```
It has insignificant impact on performance but improves readability and that's way more important here.